andela/project-jiayou-cfh
# ###137138325 Feature/UI landing page fix

#### What does this PR do?
This PR implements the change requested by the product owners and also fixes the conflict (green background) on the development branch
#### Description of Task to be completed?
- Made some changes the css to resolve the conflict
- Wrote media queries to improve the responsiveness of the site
#### How should this be manually tested?
- Test it on mobile devices
- Try to resize the page
- The unwanted green background no longer shows
#### Screenshots (if appropriate)
- Mobile phone view
<img width="216" alt="screen shot 2017-02-07 at 12 39 18 pm" src="https://cloud.githubusercontent.com/assets/24937721/22689737/9809450c-ed32-11e6-879e-4b95539555b5.png">

- IPad view
<img width="386" alt="screen shot 2017-02-07 at 12 39 57 pm" src="https://cloud.githubusercontent.com/assets/24937721/22689739/9e335616-ed32-11e6-8c60-8fdcc8c9acb5.png">
